src/Streamly/Conversion/ByteString/Lazy.hs

```diff
@@ -0,0 +1,17 @@
+module Streamly.Conversion.ByteString.Lazy where
+
+import Data.ByteString.Lazy (ByteString, foldlChunks, fromChunks)
+import Data.Word (Word8)
+import Streamly.Conversion.ByteString as Strict
+import Streamly.Internal.Memory.Array.Types (Array(..))
+import Streamly
+
+import qualified Streamly.Prelude as S
+
+fromArray :: Monad m => SerialT m (Array Word8) -> m ByteString
```

Comment:
Usually I think of the `to/from` APIs from the standpoint of  the current module. When constructing/eliminating a bytestring in the bytestring module we would omit the name bytestring and use whatever we are converting to/from in the name. If these routines were in Streamly.Prelude then we would call them toByteString/fromByteString. This gives us a consistent rule to name APIs, if there are good arguments to name them in the other way, I am open to agree to that.

Reply:
I did not notice that this is Lazy bytestring module. So I think it should be something like `toArrays` or `toArrayStream` or `toStream`  instead of `toArray`.

Reply:
Indeed that this deals with Stream of Arrays is why I wanted a different name.